content/docs/learn/typed-errors/outcome-progress.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Outcomes and in-progress
+sidebar_position: 6
+description: Describing more than success and failure.
+---
+
+# Outcomes and in-progress values
+
+Arrow Core includes three different types to model success and failure:
+`Option` when no information is available on failure,
+`Either` when the success and failure cases are disjoint,
+and `Ior` when success and failure may happen at the same time.
+But sometimes life is a bit more complicated, and these type fall short.
+Fortunately, the wonderful Kotlin community has developed libraries
+to cover other scenarios, with full integration with Arrow's `Raise` approach.
+
+## Outcomes: absence is not failure
+
+[Quiver](https://block.github.io/quiver/) introduces the
+[`Outcome`](https://block.github.io/quiver/-quiver%20-library/app.cash.quiver/index.html)
+with three states: `Present`, `Failure`, and `Absent`. The last one is what
+sets it apart from `Either`.
+
+<!--- TEST_NAME OutcomeTest -->
+
+<!--- INCLUDE
+import app.cash.quiver.*
+import app.cash.quiver.raise.*
+-->
+
+```kotlin
+val good = 3.present()
+val bad  = "problem".failure()
+val whoKnows = Absent
+```
+<!--- KNIT example-outcome-01.kt -->
+
+## In-progress values with Pedestal
+
+[Pedestal State](https://opensavvy.gitlab.io/groundwork/pedestal/api-docs/state/index.html)
+introduces [`ProgressiveOutcome`](https://opensavvy.gitlab.io/groundwork/pedestal/api-docs/state/opensavvy.state.progressive/-progressive-outcome/index.html)
+a type that combines the current state of a computation, and information
+about how the corresponding task is evolving.
+
+:::tip Success does not mean stopped
+
+The value `Success(5, Progress.loading(0.4))` is perfectly valid,
```

Comment:
`loading()` is a top-level function, I don't think you can call it as `Progress.loading()` :thinking: